src/Web/wwwroot/app/services/sideNavService.js

```diff
@@ -0,0 +1,24 @@
+angular
+    .module('bit.services')
+
+    .factory('sideNavService', function () {
+        var _service = {};
+
+        _service.toggleSideNav = function () {
+            if (localStorage.showSideNav == "false") {
```

Comment:
Cool, and sorry. I use this frequently however I wasn't sure if it was included in the project or not, and I couldn't find it but will do.